shared-libs/cht-datasource/.eslintrc.js

```diff
@@ -63,6 +63,7 @@ module.exports = {
           exemptedBy: ['inheritdoc', 'private', 'internal']
         }],
         ['jsdoc/check-tag-names']: ['error', { definedTags: ['typeParam']}],
+        ['jsdoc/empty-tags']: 0
```

Comment:
You added this because of the `fetchAndFilter` doc, right?  If so, I think it is mad that you added the docs _after_ the `@internal` tag.  If you add the text _before_ the tag, it should work fine without skipping this rule....